tests/test_foreman.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from urllib.parse import urlparse
+
+
+def test_foreman_version(api, variables):
+    if 'foreman_version' not in variables:
+        pytest.skip("'foreman_version' is not set")
+
+    status = api.resource('home').call('status')
+
+    assert status['version'] == variables['foreman_version']
```

Comment:
This is not entirely the same as the previous check. That asserted that foreman, foreman-proxy and foreman-installer all had the same version (maybe even foreman-selinux).

Perhaps we should do both? Can we find out the Smart Proxy version via the API?

Reply:
The UI shows the smart proxy version, I am not seeing that in the API. I can look into trying to test this via `testinfra` that will just require that smoker has SSH access to the server. Or we make an API update to Foreman to report the versions of these particular projects.

I think this particular test is predominantly used just to make this check for release purposes. Perhaps it belongs more so in the CI than in this test suite?